src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const object = {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add(tempResult, number) {
+      const result = tempResult + number;
```

Comment:
```suggestion
      this.result += num;
```
check all methods

Reply:
this.result doesn't work, 'this' in methods became undefined because methods use in callback function

Reply:
![image](https://github.com/mate-academy/js_advanced_calculator/assets/72886995/185f96e1-d906-4511-b576-58037717ba93)
![image](https://github.com/mate-academy/js_advanced_calculator/assets/72886995/ad18de4e-2374-44f9-a676-5a0c788065a1)